arango/batch.py

```diff
@@ -102,12 +122,22 @@ def commit(self):
         for later retrieval via its :func:`arango.batch.BatchJob.result`
         method
 
+        :returns: list of arango.batch.BatchJob or None
+
         :raises arango.exceptions.BatchExecuteError: if the batch request
             cannot be executed
         """
+
+        res = self._lock.acquire(timeout=self._batch_submit_timeout)
+
+        if not res:
+            raise BatchExecuteError("Error: Unable to reaccquire lock within time period."
```

Comment:
No need to start with `Error:` as it is already an exception message. This formatting is not consistent with other error messages.

Reply:
Okay.